ruby-monday-blog/app/views/posts/_preview.html.erb

```diff
@@ -0,0 +1,22 @@
+<% hide_tags ||= false %>
+<hr />
+<%= div_for post, class: 'post-preview' do %>
+  <%= link_to post do %>
+    <h2><%= post.title %></h2>
+  <% end %>
+  <p class="post-timestamp">Posted <%= time_ago_in_words post.created_at %>&nbsp;ago</p>
+  <p class="post-body"><%= post.body %></p>
+  <ul class="list-inline clearfix">
+    <% unless hide_tags %>
+      <% post.tags.each do |tag| %>
+        <% unless tag.content.empty? %>
```

Comment:
Well the form still sends a tag with an empty tag field. This is from dev tools:

![image](https://cloud.githubusercontent.com/assets/1369748/9206458/3c3d44b2-4068-11e5-9194-df9308a6d88d.png)